machine/views/machine_instance_views.xml

```diff
@@ -0,0 +1,196 @@
+                        </button>
+                    </div>
+                    <group>
+                        <group name="options" string="Options">
+                            <field name="name" />
+                            <field name="related_sync" invisible="1"/>
+                            <field name="parent_id" required="1" context="{'form_view_ref': 'machine.machine_instance_template_view_form', 'default_is_template': True}"/>
+                            <field name="sync"
+                                attrs="{'invisible': [('related_sync', '=', False)]}"/>
+                            <field name="is_template" invisible="1"/>
+                            <field name="partner_id"/>
+                            <field name="partner_contact_id"/>
+                            <field name="user_id"/>
+                            <field name="machine_group_ids" widget="many2many_tags" options="{'no_create_edit': True}"/>
+                            <field name="tag_ids" widget="many2many_tags" options="{'color_field': 'color', 'no_create_edit': True}"/>
+                        </group>
+                        <group name="parameters" string="Parameters">
+                            <field name="cpu_id" attrs="{'readonly': [('sync', '=', True), ('related_sync', '=', True)]}"/>
+                            <field name="os_id" attrs="{'readonly': [('sync', '=', True), ('related_sync', '=', True)]}"/>
+                            <field name="amount_storage_capacity" attrs="{'readonly': [('sync', '=', True), ('related_sync', '=', True)]}"/>
+                            <field name="amount_ram" attrs="{'readonly': [('sync', '=', True), ('related_sync', '=', True)]}"/>
+                            <field name="is_virtual" attrs="{'readonly': [('sync', '=', True), ('related_sync', '=', True)]}"/>
+                            <field name="is_container" attrs="{'readonly': [('sync', '=', True), ('related_sync', '=', True)]}"/>
+                            <field name="ip"/>
+                            <field name="domain"/>
+                        </group>
+                    </group>
+                    <notebook>
+                        <page name="change_log" string="Changes Log">
+                            <field name="change_log_ids">
+                                <tree editable="bottom">
+                                    <field name="name" attrs="{'readonly': [('id', '!=', False)]}"/>
+                                    <field name="date" attrs="{'readonly': [('id', '!=', False)]}"/>
+                                    <field name="duration" attrs="{'readonly': [('id', '!=', False)]}"/>
+                                    <field name="user_id" attrs="{'readonly': [('id', '!=', False)]}"/>
+                                    <field name="priority" attrs="{'readonly': [('id', '!=', False)]}"/>
```

Comment:
This is done for purpose that log shows what happened, so it can't be edited. But it can be removed, because sometimes we need to remove some crap logs. It is then perfectly fine to re-add log with correct info. It is intended this way and its not about security, but about usability. If we need to limit access, then thats different story (and can be done quite easily).